internal/testbed/integration/k8senrichment/README.md

```diff
@@ -15,3 +15,80 @@ on the k8s cluster with configurations as per the Dynatrace documentation page.
 Traces are generated and sent to the Collector, which then
 exports to the test where the k8s attributes are asserted on the
 received traces.
+
+## Running the tests locally
+
+To execute the tests locally, follow these steps:
+
+### Start a `kind` cluster
+
+To create a new cluster, execute the following command:
+
+```shell
+kind create cluster
+```
+### Build the collector binary
+
+This is done using the 
+```shell
+make build
+```
+command. This will build the collector distro, and place the built binary 
```

Comment:
I suggest putting this inline if it's inside a sentence.
```suggestion
This is done using the `make build` command.
This will build the collector distro, and place the built binary 
```